patches/chrome-browser-enterprise-webstore-features.cc.patch

```diff
@@ -0,0 +1,15 @@
+diff --git a/chrome/browser/enterprise/webstore/features.cc b/chrome/browser/enterprise/webstore/features.cc
+index a66c87787498ba66e7ea09b89f1bfee20158d91b..1e631d7d6689e4adfe26609d92b7ae4bf06421eb 100644
+--- a/chrome/browser/enterprise/webstore/features.cc
++++ b/chrome/browser/enterprise/webstore/features.cc
+@@ -8,7 +8,8 @@
+ 
+ namespace enterprise::webstore {
+ 
+-// Controls whether webstore navigation throttle is enabled.
+-BASE_DECLARE_FEATURE(kChromeWebStoreNavigationThrottle);
++BASE_FEATURE(kChromeWebStoreNavigationThrottle,
```

Comment:
I'm confused, is this an upstream patch?

Reply:
I am hoping this will be an upstream patch, for now I just left a comment in the original CL here: https://chromium-review.googlesource.com/c/chromium/src/+/6319096/comment/63993a26_7b78a891/